pmm-app/src/pmm-qan/panel/components/Details/Metadata/Metadata.styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { getPmmTheme } from 'shared/components/helpers/getPmmTheme';
+
+export const getStyles = (theme: GrafanaTheme2) => {
+  const parameters = getPmmTheme(theme.v1);
+
+  return {
+    metadataColumn: css`
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      word-break: keep-all;
+      color: ${parameters.mainTextColor} !important;
```

Comment:
can we avoid `!important`?

Reply:
these will be inline styles, I don't really understand why it is needed.

Reply:
Yes, we can avoid this, but in this case I've kept the style of the table 'Metrics' above.